.github/workflows/pre-merge-ci.yaml

```diff
@@ -25,6 +25,14 @@ jobs:
     - name: Run checks
       run: make ci
 
+    - name: Validate added bundles
+      run: |
+        set -o errexit
+        set -o pipefail
+        set -o nounset
```

Comment:
Not sure we need this, doesn't hurt, so nitpickery

Reply:
Removed it. Thanks